exonum-light-client/src/main/java/com/exonum/client/ExonumClient.java

```diff
@@ -77,6 +80,95 @@
    */
   Optional<TransactionResponse> getTransaction(HashCode id);
 
+  /**
+   * Returns the <em>blockchain height</em> which is the height of the latest committed block
+   * in the blockchain. The block height is a distance between the last block
+   * and the "genesis", or initial, block. Therefore, the blockchain height is equal to the number
+   * of blocks plus one.
+   *
+   * <p>For example, the "genesis" block has height {@code h = 0}. The latest committed block
+   * has height {@code h = getBlockHashes().size() - 1}.
+   *
+   * @throws RuntimeException if the client is unable to complete a request
+   *        (e.g., in case of connectivity problems)
+   */
+  long getBlockchainHeight();
+
+  /**
+   * Returns the information about the block with transaction hashes included at this block.
+   * @param height blockchain height starting from 0 (genesis block)
+   * @return block information response
+   * @throws RuntimeException if the client is unable to complete a request
+   *        (e.g., in case of connectivity problems)
+   * @throws RuntimeException if block is not found by the requested height,
+   *        i.e. the requested height is greater then actual blockchain height
+   * @throws IllegalArgumentException if the given height is negative
+   */
+  BlockResponse getBlockByHeight(long height);
+
+  /**
+   * Returns blockchain blocks information for the requested range.
+   * @param count Number of blocks to return.
+   *        It should not be negative and should not be greater then
+   *        It should be in range [1, {@linkplain ExonumApi#MAX_BLOCKS_PER_REQUEST}]
+   * @param skipEmpty if {@code true}, then only non-empty blocks will be returned
+   * @param heightMax maximum height of the returned blocks. The blocks are returned
+   *        in reverse order, starting from the <b>heightMax</b> and
+   *        at least up to the <b>heightMax - count + 1</b>.
```

Comment:
"at least up to the <b>heightMax - count + 1</b>."
— I don't find this wording easy to understand. at least down to? @Maria-Nosyk What would you suggest?

Reply:
Also, bold seems too much.